core-services/prow/02_config/redhat-developer/intellij-openshift-connector/_prowconfig.yaml

```diff
@@ -0,0 +1,12 @@
+tide:
+  queries:
+  - labels:
+    - lgtm
```

Comment:
No `approved` label needed here?

Reply:
@danilo-gemoli : please add if necessary, not sure to understand correctly this coinfig file.

Reply:
It really depends by the workflow you're using. Such a configuration allows Tide to merge PRs as long as they have the `lgtm` label. If you're ok, that's it.

Reply:
This will mean that anyone from the redhat-developer organization can merge a PR by just adding the lgtm. Is this what you need?

Reply:
yes, we want a real person, one of the approver above to merge the PRs